example/windows/PickerExample/PickerExample.vcxproj

```diff
@@ -1,25 +1,28 @@
-﻿<?xml version="1.0" encoding="utf-8"?>
-<Project ToolsVersion="15.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
-  <Import Project="..\packages\Microsoft.Windows.CppWinRT.2.0.190730.2\build\native\Microsoft.Windows.CppWinRT.props" Condition="Exists('..\packages\Microsoft.Windows.CppWinRT.2.0.190730.2\build\native\Microsoft.Windows.CppWinRT.props')" />
+<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="16.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Import Project="..\packages\Microsoft.Windows.CppWinRT.2.0.210312.4\build\native\Microsoft.Windows.CppWinRT.props" Condition="Exists('..\packages\Microsoft.Windows.CppWinRT.2.0.210312.4\build\native\Microsoft.Windows.CppWinRT.props')" />
+  <Import Project="$(SolutionDir)\ExperimentalFeatures.props" Condition="Exists('$(SolutionDir)\ExperimentalFeatures.props')" />
   <PropertyGroup Label="Globals">
     <CppWinRTOptimized>true</CppWinRTOptimized>
     <CppWinRTRootNamespaceAutoMerge>true</CppWinRTRootNamespaceAutoMerge>
     <MinimalCoreWin>true</MinimalCoreWin>
-    <ProjectGuid>{2c471b95-b391-4769-b397-a3b1ba7d9e99}</ProjectGuid>
+    <ProjectGuid>{99c4cfd3-bdaf-42ae-934a-8c79ae2a66e0}</ProjectGuid>
     <ProjectName>PickerExample</ProjectName>
     <RootNamespace>PickerExample</RootNamespace>
     <DefaultLanguage>en-US</DefaultLanguage>
-    <MinimumVisualStudioVersion>15.0</MinimumVisualStudioVersion>
+    <MinimumVisualStudioVersion>16.0</MinimumVisualStudioVersion>
     <AppContainerApplication>true</AppContainerApplication>
     <ApplicationType>Windows Store</ApplicationType>
     <ApplicationTypeRevision>10.0</ApplicationTypeRevision>
     <WindowsTargetPlatformVersion Condition=" '$(WindowsTargetPlatformVersion)' == '' ">10.0.18362.0</WindowsTargetPlatformVersion>
-    <WindowsTargetPlatformMinVersion>10.0.15063.0</WindowsTargetPlatformMinVersion>
-    <PackageCertificateKeyFile>PickerExample_TemporaryKey.pfx</PackageCertificateKeyFile>
-    <PackageCertificateThumbprint>38A588375FC02AA1285287D44A45D3C92F010DAD</PackageCertificateThumbprint>
-    <PackageCertificatePassword>password</PackageCertificatePassword>
+    <WindowsTargetPlatformMinVersion>10.0.16299.0</WindowsTargetPlatformMinVersion>
+    <AppxPackageSigningEnabled>false</AppxPackageSigningEnabled>
```

Comment:
This is needed to avoid using a temp cert that will expire. Normally it breaks the ability to deploy release builds using `run-windows`, but our CLI deployment code is already having trouble understanding the repo layout here, even with hints. Visual Studio will deploy correctly in both debug and release just fine with this set.

Reply:
Does this mean the cli doesn't work with this project? That seems concerning

Reply:
Yeah, the CLI does not seem to work to deploy the example app. This would not affect apps consuming the module outside the repo. It can build, but then deploy looks for the AppX in the wrong place. There is a `--root` option, but that seemed to affect other bits in the build indirectly. I remember CLI deploy for non-standard directory structures came up before, and IIRC it was something we didn't care to invest time in fixing?

Reply:
@jonthysell

Reply:
The underlying bug is I think in CLI deploy code, where it assumes AppX location relative to npm package root instead of based on where solution/project is.